### PR TITLE
patches (N50: sm64jsarchive #1474)

### PR DESCRIPTION
**Pull request by uuphoria2**
forgot to patch these up! sorry :P

(n50 sm64jsarchive #1474 is what im making the patch for)

### CODE REVIEW

**Comment by uuphoria2** [comment]
@janhohenheim Merge this whenever you can. 👍

**Comment by uuphoria2** [comment]
@janhohenheim Merge this whenever you can, please 😄

File: content/news/050/index.md [hunk 1]
```diff
@@ -103,21 +103,20 @@ For the full changelog, see the [release notes][wor-release-notes].
...
 
-### [SM64JSARCHIVE][sm64jsarchive]
+### [sm64jsarchive][sm64jsarchive]
 
-![Super Mario 64 JavaScript](sm64jsarchive.jpg)
+![Super Mario 64 JavaScript Archive](sm64jsarchive.jpg)
 
 [SM64JSARCHIVE][sm64jsarchive] is an actively maintained fork of [sm64js]: a decompilation project of Super Mario 64 to JavaScript.
-Additional sidenote: The mmo servers arent always running for the game
+Additional sidenote: The MMO servers arent always running for sm64jsarchive.
```
Comment (Vrixyz) [inline content/news/050/index.md:111]:
```suggestion
Additional sidenote: The MMO servers are not always running for sm64jsarchive.
```

**Review by Vrixyz — APPROVED** [review]
small nitpick but looks good to me.

**Comment by janhohenheim** [comment]
Thanks for the patches :)

**Comment by uuphoria2** [comment]
> small nitpick but looks good to me.

thanks for catching that! i didnt notice it

**Comment by uuphoria2** [comment]
> Thanks for the patches :)

thanks for merging this! :D

**Pull request merged**